smarts/core/sensors.py

```diff
@@ -992,7 +992,10 @@ def lazy_calculate_waypoints():
         if self._mission_planner.mission.task is not None:
             if isinstance(self._mission_planner.mission.task, UTurn):
                 waypoints_with_task = self._mission_planner.uturn_waypoints(
-                    self._sim, self._vehicle.pose, self._vehicle
+                    self._sim,
+                    self._vehicle.pose,
+                    self._vehicle,
+                    self._mission_planner.mission.task.initial_speed,
```

Comment:
line 1005, the `lazy_calculate_waypoints` is not used in `cut_in_waypoints`, do you minding removing the param?